schema/spacewalk/common/tables/suseClusters.sql

```diff
@@ -0,0 +1,48 @@
+-- Copyright (c) 2020 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE suseClusters (
+    id          NUMERIC NOT NULL
+                    CONSTRAINT suse_clusters_id_pk PRIMARY KEY,
+    org_id      NUMERIC NOT NULL
+                    CONSTRAINT rhn_server_oid_fk
+                    REFERENCES web_customer (id)
+                    ON DELETE CASCADE,
+    label       VARCHAR(256) NOT NULL,
+    name        VARCHAR(256) NOT NULL,
+    description VARCHAR(4096) NOT NULL DEFAULT '',
```

Comment:
@mcalmer I think you're thinking about Oracle, where `null` and an empty string is indeed interpreted the same, but not in PostgreSQL, so this is technically valid.
But still, since it's a DB specific and generally confusing, we can just make it nullable instead.

Reply:
Maybe it was the other way around? As it was in oracle the same, they tried to prevent that somebody use it in postgresql and therefor they setup a check which does not allow it.

Reply:
I changed to ` VARCHAR(4096) DEFAULT ''`. I tested and PostgreSQL treats doesn't treat empty string as `NULL` like Oracle.